File: backend/connectors/location-monitoring-connector/src/main/java/ru/nord/siwatch/backend/connectors/locationmonitoring/models/LocationInfo.java
package ru.nord.siwatch.backend.connectors.locationmonitoring.models;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class LocationInfo {
    private String deviceId;
    private LocalDateTime deviceTime;
    private double latitude;
    private double longitude;
    private double altitude;
    private double speed;
    private double direction;
    private double accuracy;
}

File: watch-app/SiWatchApp/Models/LocationInfo.cs
using Tizen.Location;

namespace SiWatchApp.Models
{
    public class LocationInfo : PositionInfo
    {
        public double Accuracy { get; set; }

        public LocationInfo(Location location)
        {
            Latitude = location.Latitude;
            Longitude = location.Longitude;
            Altitude = location.Altitude;
            Accuracy = location.Accuracy;
        }

        public override string ToString()
        {
            return $"LocationInfo{{Latitude={Latitude},Longitude={Longitude},Altitude={Altitude},Accuracy={Accuracy}}}";
        }
    }
}
